src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -82,6 +82,8 @@
 
     <h2>Translation Messages</h2>
 
+    {% block translation_messages %}
```

Comment:
As we are in a template managing templates, what about just `messages`? Would that be clear enough?